packages/api/src/api/whispers/WhispersApi.ts

```diff
@@ -0,0 +1,50 @@
+import type { UserIdResolvable } from '@twurple/common';
+import { extractUserId, rtfm } from '@twurple/common';
+import { BaseApi } from '../BaseApi';
+
+/**
+ * The API methods that deal with whispers.
+ *
+ * Can be accessed using 'client.whispers' on an {@ApiClient} instance
+ *
+ * ## Example
+ * ```ts
+ * const api = new ApiClient(new StaticAuthProvider(clientId, accessToken));
+ * await api.whispers.sendWhisper('61369223', '86753099', 'Howdy, partner!');
+ * ```
+ */
+
+@rtfm('api', 'WhispersApi')
+export class WhispersApi extends BaseApi {
```

Comment:
Should be `HelixWhisperApi` to conform with the rest of the naming, as it's a Helix API and these class names generally use the singular form.

Also should be in the `helix` folder.